### ISSUE TITLE
Can't extract energy from Flux Point

### ISSUE THREAD

**Issue by Wlnfr** [CyclopsMC/IntegratedTunnels#237]
#### Issue type:

- :bug: Bug 

____

#### Short description:
New energy networks(any combination of interfaces, importers and exporters) stopped working with Flux Point from Flux Networks. It just doesn't pull any energy from it regardless of how I try. It did work some time in the past since I still have the working network in my world

#### Steps to reproduce the problem:

1. Create a new world
2. Place any source of FE
3. Connect it to Flux Plug and create a Flux Network
4. Place a Flux Point and connect it to created Flux Network
5. Place an energy interface on Flux point
6. Place a battery from Integrated Dynamics(or any FE acceptor)
7. Place an energy exporter on a battery, connect it to energy interface and add a variable card for it to become active

#### Expected behaviour:
Battery starts filling up

____

#### Versions:

- Minecraft: 1.16.5
- Forge: 36.2.2
- Mod list:
CommonCapabilities-1.16.5-2.7.0
CyclopsCore-1.16.5-1.11.8
FluxNetworks-1.16.5-6.1.7.12
IntegratedDynamics-1.16.5-1.9.2
IntegratedTunnels-1.16.5-1.8.5
jei-1.16.5-7.7.1.118
theoneprobe-1.16-3.1.4
topaddons-1.16.5-2.1.5-beta

Screenshots:
![2021-08-11_18 05 47](https://user-images.githubusercontent.com/8050905/129057221-ba379c33-d85e-4f1e-9be6-202ca101c21f.png)

This setup for reasons unknown to me worked:
![2021-08-11_18 21 40](https://user-images.githubusercontent.com/8050905/129057547-4711769b-1b83-431e-a2a1-43ec98a2ba88.png)

**Comment by rubensworks** [comment]
Thanks for reporting!

**Comment by rubensworks** [comment]
> It did work some time in the past since I still have the working network in my world

What time period are you talking about?
Nothing has changed regarding how these parts work for a long time, so I would be surprised this is caused by something on IT's end.

I don't know how this Flux Networks mod works myself, but are you sure that its setup is valid? Can you extract/insert energy from other mods into those Flux Networks? And what happens if you let IT interact with other energy mods?

**Comment by Wlnfr** [comment]
> What time period are you talking about?

World was created at June 17th, I made my flux network maybe within a week or so and it did work correctly at that time.

> Nothing has changed regarding how these parts work for a long time, so I would be surprised this is caused by something on IT's end.

I am also a bit confused, since as my testing showed, the issue exists in a barebones Flux Networks and IT(plus its dependencies) modpack, and Flux Networks were last updated April 22th, so it definitely didn't change anything and it doesn't have dependencies either, which leaves us with IT, ID, Common Capabilities, CyclopsCore and Forge(When I started my world I was using 36.1.16 Forge) as suspects. The problem is, I already shuffled through different versions of all these mods and forge but still can't make it work.

> I don't know how this Flux Networks mod works myself, but are you sure that its setup is valid? Can you extract/insert energy from other mods into those Flux Networks? And what happens if you let IT interact with other energy mods?

The setup is valid and it can be seen on first screenshot, a small cube with blue corners is connected directly to ID battery(the ones on the left) and filled it up. FN also accepts energy both from IT(first screenshot, big cube with blue outline and three batteries on the right which are full) and other energy mods(I tested Woot's Creative energy source and Thermal Expansion's dynamos) without any problems directly or with IT in-between. FN can also directly output fine to any FE acceptor(ID battery, Woot machines, Thermal Expansion machines), but IT can't extract energy from FN in any way

**Comment by rubensworks** [comment]
Do you know if the FN extractors have an internal buffer? If not, then that may explain why things are not working as expected.

Could you place a Machine Reader on a FN extractor to see what FE things it exposes?

**Comment by Wlnfr** [comment]
Ignore post above. I found the problem
Both FN an IT have a default energy transfer rate limits IT=1000 FE/t and FN=800000 FE/t. IT doesn't want to pull energy from FN when its(IT's) limit is lower than FN's. the moment I increased IT's limit to be above FN(just by adding three zeros) it immediately started working, it seem I have done that to my original working network in the world and forgot about it. 
TLDR: Everything will work if IT's energy transfer limit rate =>(equals or more than) FN's energy transfer limit rate

**Comment by Wlnfr** [comment]
![2021-08-12_15 45 10](https://user-images.githubusercontent.com/8050905/129198983-9a1b2634-eb09-46ba-b841-26566a806763.png)
![2021-08-12_15 45 14](https://user-images.githubusercontent.com/8050905/129199000-38be7bae-1ae1-4b0a-a686-68aaba1c05b6.png)
![2021-08-12_15 45 22](https://user-images.githubusercontent.com/8050905/129199015-19a094e8-b545-413b-97d6-74e0095a7a1f.png)
![2021-08-12_15 45 26](https://user-images.githubusercontent.com/8050905/129199031-1dc69e67-225a-4c6d-b48f-fc33ddddaf4d.png)

**Comment by rubensworks** [comment]
Ah, good to hear the problem is solved.

> IT doesn't want to pull energy from FN when its(IT's) limit is lower than FN's.

IT _should_ pull energy "up to the given limit".
So if the source contains less than the limit, it should still pull it.
Perhaps FN is considering the transfer limit to also be a lower bound?

**Comment by Wlnfr** [comment]
I have no idea how FN implemented it, but since this seems like FN issue, should I make an issue on its tracker?

**Comment by Wlnfr** [comment]
Additional info:
As it turns out, my original network has default transfer rate(1000 FE/t). same with FN(800000 FE/t) and it still works, but only because on the other side of flux point there is a controller(FE acceptor) from Refined Storage and IT can pull energy from FN as long as controller is there
![2021-08-12_16 07 11](https://user-images.githubusercontent.com/8050905/129202584-17fdfbae-ad1d-4f85-9967-878828c64eb1.png)

**Comment by rubensworks** [comment]
> I have no idea how FN implemented it, but since this seems like FN issue, should I make an issue on its tracker?

Makes sense, thanks!

Closing this one here in the meantime.
